cli/src/cmd/forge/verify/sourcify.rs

```diff
@@ -88,7 +88,7 @@ impl VerificationProvider for SourcifyVerificationProvider {
                         SimpleCast::checksum_address(&args.address)?
                     );
                     let response = client
-                        .post(SOURCIFY_URL)
+                        .post(args.verifier.verifier_url.as_ref().unwrap_or(&SOURCIFY_URL.to_owned()))
```

Comment:
I believe `.as_deref` should work
```suggestion
                        .post(args.verifier.verifier_url.as_deref().unwrap_or(SOURCIFY_URL))
```

Reply:
fixed